README.md

```diff
@@ -43,22 +48,119 @@ BBLAYERS += "${BSPDIR}/sources/meta-cyclonedx"
+compatibility with tools that only support CycloneDX 1.4, you can configure:
+
+```sh
+CYCLONEDX_SPEC_VERSION = "1.4"
+```
+
+**Version differences:**
+- **1.4**: Legacy format for compatibility with older tools
+- **1.6**: Modern format with component scopes, enhanced metadata, and timestamps (default)
+
+### Runtime vs Build-time Packages
+
 By default, meta-cyclonedx will only include run-time packages in the SBOM,
 which drastically reduces the number of potentially irrelevant packages.
 However, this can lead to valid packages being omitted from the SBOM
 (see [here](#potentially-missing-packages-after-runtime-filtering)).
+
 If preferred, you can add the following configuration setting
 (e.g in your local.conf), which will cause meta-cyclonedx to include
 all build-time packages as well:
 
+```sh
+CYCLONEDX_RUNTIME_PACKAGES_ONLY = "0"
+```
+
+### Component Scopes
+
+When including both runtime and build-time packages, meta-cyclonedx uses
+[CycloneDX component scopes](https://cyclonedx.org/docs/1.6/json/#components_items_scope)
+to differentiate between them:
+
+- Runtime packages are marked with `"scope": "required"`
+- Build-time only packages are marked with `"scope": "excluded"`
+
+This allows tools to filter components based on their use case:
+
+- **CVE matching**: Focus on components with `"scope": "required"`
+- **License compliance**: Include all components regardless of scope
+- **Supply chain tracking**: Include all components regardless of scope
+
+### Component Scopes (CycloneDX 1.6)
+
+Component scopes are enabled by default when using CycloneDX 1.6. If you need to
+disable them (e.g., for compatibility with certain SBOM profiles or tools):
+
+```sh
+CYCLONEDX_ADD_COMPONENT_SCOPES = "0"
 ```
+
+**Note**: Component scopes are only available in CycloneDX 1.6. This setting has
+no effect when `CYCLONEDX_SPEC_VERSION = "1.4"`.
+
+### Vulnerability Analysis Timestamps (CycloneDX 1.6)
+
+By default, vulnerability analysis records include `firstIssued` and `lastUpdated`
+timestamps in CycloneDX 1.6. To generate minimal VEX documents without timestamps:
+
+```sh
+CYCLONEDX_ADD_VULN_TIMESTAMPS = "0"
+```
+
+### Advanced Configuration Summary
+
+```sh
+# Specification version (default: "1.6")
+CYCLONEDX_SPEC_VERSION = "1.6"  # or "1.4"
+
+# Include build-time packages (default: "0" = runtime only)
 CYCLONEDX_RUNTIME_PACKAGES_ONLY = "0"
+
+# Add component scopes in 1.6 (default: "1")
+CYCLONEDX_ADD_COMPONENT_SCOPES = "1"
+
+# Add vulnerability timestamps in 1.6 (default: "1")
+CYCLONEDX_ADD_VULN_TIMESTAMPS = "1"
+```
+
+### SBOM Profile Compatibility
+
+Different organizations may have specific SBOM profile requirements. The configuration
+options above allow you to customize the generated SBOMs to meet various profile needs:
+
+**For profiles that prohibit component scopes:**
+```sh
+CYCLONEDX_ADD_COMPONENT_SCOPES = "0"
+```
+
+**For profiles requiring minimal metadata:**
```

Comment:
semi-related, but if we already talk about adding documentation on "minimal metadata", would it also make sense to add config flags to other features, such as dependencies and licenses, which are currently always enabled?

Reply:
agreed, let's open a separate issue on this.

I just brought it up since you are adding a documentation section that explicitly mentions "minimal metadata".

I suggest we remove this particular wording from the README in this PR until we decided what " minimal metadata" consists of and have consistent feature flags in place for that.